flow/flow.go

```diff
@@ -770,6 +890,10 @@ func (f *Flow) GetFieldInt64(field string) (_ int64, err error) {
 		return f.Metric.GetFieldInt64(fields[1])
 	case "LastUpdateMetric":
 		return f.LastUpdateMetric.GetFieldInt64(fields[1])
+	case "TCPFlowMetric":
+		if f.TCPFlowMetric != nil {
+			return f.TCPFlowMetric.GetFieldInt64(fields[1])
```

Comment:
you can call directly the return f.TCPFlowMetric.GetFieldInt64( ....  as you do the test already line 794